bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -810,6 +810,23 @@ impl Room {
 
         Ok(())
     }
+
+    /// Attempt to manually resend messages that failed to send due to issues
+    /// that should now have been fixed. This is useful for example, when
+    /// there's a
+    /// `SessionRecipientCollectionError::VerifiedUserChangedIdentity` error;
+    /// the user may have re-verified on a different device and would now
+    /// like to send the failed message that's waiting on this device.
+    ///
+    /// # Arguments
+    ///
+    /// * `transaction_id` - The send queue transaction identifier of the local
+    ///  echo the send error applies to.
+    pub async fn resend(&self, transaction_id: String) -> Result<(), ClientError> {
+        let transaction_id: OwnedTransactionId = transaction_id.into();
+        self.inner.send_queue().unwedge(&transaction_id).await?;
```

Comment:
```suggestion
        let transaction_id: &TransactionId = transaction_id.into();
        self.inner.send_queue().unwedge(transaction_id).await?;
```

Reply:
Ah nice, didn't know you could `into` a borrowed type.

Reply:
I had to add an `as_str()` to this to make the compiler happy.